src/block.js

```diff
@@ -17,14 +17,17 @@ export function init (elem, {normalize, shouldSpellcheck}) {
   elem.classList.add(config.editableClass)
 
   if (normalize) content.tidyHtml(elem)
+  if (plainText) content.normalizePlainText(elem)
```

Comment:
I think you don't use `getPlainText()` in `extractContent()` yet. Is this an oversight or on purpose?

I thin it would make sense to normalize on init like you do here and every time the framework reads the content into its model where `extractContent()` is used.